src/main/java/com/nha/abdm/wrapper/hiu/facade/consent/HIUFacadeConsentController.java

```diff
@@ -0,0 +1,43 @@
+/* (C) 2024 */
+package com.nha.abdm.wrapper.hiu.facade.consent;
+
+import com.nha.abdm.wrapper.common.responses.FacadeResponse;
+import com.nha.abdm.wrapper.hiu.hrp.consent.HIUConsentInterface;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.ConsentStatusRequest;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.FetchConsentRequest;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.InitConsentRequest;
+import com.nha.abdm.wrapper.hiu.hrp.consent.requests.OnNotifyRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HIUFacadeConsentController {
+
+  @Autowired private HIUConsentInterface hiuConsentInterface;
+
+  @PostMapping({"/consent-init"})
+  public ResponseEntity<FacadeResponse> initiateConsentRequest(
+      @RequestBody InitConsentRequest initConsentRequest) {
+    return hiuConsentInterface.initiateConsentRequest(initConsentRequest);
+  }
+
+  @PostMapping({"/consent-status"})
+  public ResponseEntity<FacadeResponse> consentRequestStatus(
```

Comment:
If we are making the API synchronous, then the response of this API should be containing `consentStatus` details. Correct me if I am wrong.

Reply:
Added an issue to handle this: https://github.com/NHA-ABDM/ABDM-wrapper/issues/45